docker/volumes/api/kong.yml

```diff
@@ -153,10 +153,18 @@ services:
           - /storage/v1/
     plugins:
       - name: cors
-
+  ## Analytics routes
```

Comment:
we should remove changes to kong and directly call the docker service hostname, this does not reflect what is in cli, since analytics routes are not on platform yet.